_episodes/06-first_session_end.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "End of first session"
```

Comment:
We should be able to use the `break` layout for this instead of a lesson.

See what they did in the intro course https://github.com/ImperialCollegeLondon/introductory_grad_school_git_course/blob/gh-pages/_episodes/l1-03-break.md?plain=1

Reply:
That looks so much better! Thanks @AdrianDAlessandro!